packages/hoppscotch-cli/README.md

```diff
@@ -24,13 +24,24 @@ hopp [options or commands] arguments
 
    - Displays the help text
 
-3. #### **`hopp test <file_path>`**
+3. #### **`hopp test [options] <file_path>`**
    - Interactive CLI to accept Hoppscotch collection JSON path
    - Parses the collection JSON and executes each requests
    - Executes pre-request script.
    - Outputs the response of each request.
    - Executes and outputs test-script response.
 
+    #### Options:
+    ##### `hopp test -e / --env <file_path> test <file_path>`
+    - Accepts path to env.json with contents in below format:   
+        ```json
+        {
+            "ENV1":"value1",
+            "ENV2":"value2"
+        }
+        ```
+    - Applies ENVs from env.json file as selected env variables.
```

Comment:
This is a more technical internal aspect of how we are loading in env variables.
Documentation is user level, we don't need to talk about technical details in user docs.
Instead, explain how the user can access it.

```suggestion
    - You can now access those variables using `pw.env.get('<var_name>')`
      Taking the above example, `pw.env.get("ENV1")` will return `"value1"`
```